fendermint/app/options/src/materializer.rs

```diff
@@ -67,3 +69,20 @@ pub struct MaterializerRemoveArgs {
     #[arg(long, short)]
     pub testnet_id: TestnetId,
 }
+
+#[derive(Args, Debug)]
+pub struct MaterializerImportKeyArgs {
+    /// Path to the manifest file.
+    ///
+    /// This is used to determine the testnet ID as well as to check that the account exists.
+    #[arg(long, short)]
+    pub manifest_file: PathBuf,
```

Comment:
Is this mandatory? Could it use the default manifest file (there exists such a thing, right?) if this argument isn't provided?

Reply:
There is no default manifest. I think it's not a bad sanity check that what you are trying to do makes sense.